oidc/client.go

```diff
@@ -763,7 +763,7 @@ func (c *Client) RefreshToken(refreshToken string) (jose.JWT, error) {
 		return jose.JWT{}, err
 	}
 
-	jwt, err := jose.ParseJWT(t.IDToken)
+	jwt, err := jose.ParseJWT(t.AccessToken)
```

Comment:
@ericchiang I am fine leaving it with IDToken, since it seems most providers return that when `grant_type` is `refresh_token`.  I just need to go back to the OpenID provider Im looking at using and have them change their response as they don't return the `id_token`.